appveyor.yml

```diff
@@ -5,6 +5,7 @@ version: '{build}'
 clone_depth: 50
 
 environment:
+  OPENSSL_VER: 1_1_0g
```

Comment:
This will easily break when a new openssl release is published, as slproweb.com will only offer the most recent version for download.

Reply:
Yes it is a problem. But I have copied your code and the link for download the `xidel` utility is broken. 
This is another link: https://netix.dl.sourceforge.net/project/videlibri/Xidel/Xidel%200.9.6/xidel-0.9.6.win32.zip